docs/2.8.0/docs/canton/usermanual/kms/external_key_storage_aws.rst

```diff
@@ -0,0 +1,199 @@
+be aware that using the key id is not enough and we are required to register the key using its `ARN`.
+
+Here is a brief reminder of the initial keys that each node owns:
+
++-----------------------------------+------------+--------------------+---------------+--------------+-----------------+
+| **Keys**                          | **Domain** | **Domain-Manager** | **Sequencer** | **Mediator** | **Participant** |
++===================================+============+====================+===============+==============+=================+
+| `(Signing) Namespace Key`         | x          | x                  |               |              | x               |
++-----------------------------------+------------+--------------------+---------------+--------------+-----------------+
+| `Signing Key`                     | x          | x                  | x             | x            | x               |
++-----------------------------------+------------+--------------------+---------------+--------------+-----------------+
+| `(Asymmetric) Encryption Key`     |            |                    |               |              | x               |
++-----------------------------------+------------+--------------------+---------------+--------------+-----------------+
+
+Depending on the key purpose and the default signing and encryption schemes defined in Canton,
+you need to pre-generate the corresponding AWS KMS keys with the correct settings:
+
++-------------------+-----------------------------------------------------------------------+-----------------------------------------------------+
+| **Provider**      | **SIGNING**                                                           | **ENCRYPTION**                                      |
++===================+=======================================================================+=====================================================+
+| AWS               | - **Key Purpose:** `SIGN_VERIFY`                                      | - **Key Purpose:** `ENCRYPT_DECRYPT`                |
+|                   | - **Key Algorithms:** `ECC_NIST_P256` or `ECC_NIST_P384`              | - **Key Algorithm:** `RSA_2048`                     |
++-------------------+-----------------------------------------------------------------------+-----------------------------------------------------+
+
+Finally, using the previously registered keys, **we need to initialize our nodes**:
+
+- :ref:`initialize domain <manually-init-domain>`
+- :ref:`initialize participants <manually-init-participant>`
+
+Please be aware that for this specific use-case **we must use the keys that we registered and not generate new ones**.
+In other words, registering the keys replaces the `Keys Initialization` step in the guide to :ref:`manually initialize a node <manually_initializing_node>`.
+
+.. _participant_aws_kms_migration:
+
+Participant Node Migration to KMS Crypto Provider
```

Comment:
This section and the one after (manual key rotation) seem to be exactly the same for AWS and GCP. I think those we can abstract away to their own page to avoid repeating